fuel-base/src/main/kotlin/fuel/RealHttpLoader.kt

```diff
@@ -6,48 +6,108 @@ import okhttp3.Call
 import okhttp3.Request.Builder
 import okhttp3.Response
 
+private enum class Method {
```

Comment:
I believe we should drop this class for strings. See reasons in another comments

Reply:
Ok, will update this

Reply:
Done!